src/cachemachine/schemas/post.json

```diff
@@ -77,6 +86,15 @@
                         "num_weeklies": 2,
                         "num_dailies": 3
                     },
+                    {
```

Comment:
If we fold this into the same class this diff will change and be a lot smaller, just adding cycle

Reply:
Agreed.